src/sentry/eventstream/kafka/backend.py

```diff
@@ -190,7 +189,7 @@ def commit_offsets():
 
                 task_kwargs = get_task_kwargs_for_message(message.value())
```

Comment:
This will perform the `skip_consume` checks: https://github.com/getsentry/sentry/blob/5628ab61b50d62ca2a39d7ffb9e68633f8435b08/src/sentry/eventstream/kafka/protocol.py#L23-L24

I don't recall if there was a specific reason we did this at the protocol level (e.g. if these messages couldn't be decoded for some reason into an `Event` object) so I took the conservative approach here and left it alone.